app/views/tags/edit.html.erb

```diff
@@ -13,14 +13,9 @@
                  },
                  class: "app-c-contextual-guidance__form" do %>
       <% @edition.document_type.tags.each do |tag_field| %>
-        <% if @edition.access_limit.present? && tag_field.id == "primary_publishing_organisation" %>
-          <%= render "govuk_publishing_components/components/warning_text", {
-            text: t("tags.edit.organisation_warning"),
-          } %>
-        <% end %>
-
         <div id="<%= tag_field.id %>">
-          <%= render "tags/tags/#{tag_field.type}_input",
+          <%= render "fields/edit/#{tag_field.id}",
+            edition: @edition,
             tag_field: tag_field,
             tags: params[:tags] || @edition.tags,
             issues: @issues %>
```

Comment:
I don't think this should have tag_field or tags passed in as it looks like a coupling of this to the tag view and a [deviation from the approach used for content](https://github.com/alphagov/content-publisher/blob/ab212f07bfcdcf10678a33479e8d7a5aa24994dc/app/views/content/edit.html.erb#L21-L23).

Can this match the interface used for content?